modules/telemetry-what-information-is-collected.adoc

```diff
@@ -12,28 +12,35 @@ endif::[]
 
 The following information is collected by Telemetry:
 
-* The unique random identifier that is generated during an installation
-* Version information, including the {product-title} cluster version and installed update details that are used to determine update version availability
-* Update information, including the number of updates available per cluster, the channel and image repository used for an update, update progress information, and the number of errors that occur in an update
-* The name of the provider platform that {product-title} is deployed on and the data center location
-* Sizing information about clusters, machine types, and machines, including the number of CPU cores and the amount of RAM used for each
+**System Information**
+
+* Version information, including the {product-title} cluster version and installed update details that are used to determine update version availability.
```

Comment:
Because the list items are not complete sentences, do not use any end punctuation per the ISG.
```suggestion
* Version information, including the {product-title} cluster version and installed update details that are used to determine update version availability
```

Applies to all list items